src/Compiler/FSComp.txt

```diff
@@ -1731,4 +1731,5 @@ featureUnmanagedConstraintCsharpInterop,"Interop between C#'s and F#'s unmanaged
 3584,tcDotLambdaAtNotSupportedExpression,"Shorthand lambda syntax is only supported for atomic expressions, such as method, property, field or indexer on the implied '_' argument. For example: 'let f = _.Length'."
 3855,tcNoStaticMemberFoundForOverride,"No static abstract member was found that corresponds to this override"
 3859,tcNoStaticPropertyFoundForOverride,"No static abstract property was found that corresponds to this override"
-3860,chkStaticMembersOnObjectExpressions,"Object expressions cannot implement interfaces with static abstract members or declare static members."
\ No newline at end of file
+3860,chkStaticMembersOnObjectExpressions,"Object expressions cannot implement interfaces with static abstract members or declare static members."
+3861,chkStaticMembersDirectlyOnInterfaces,"Interfaces cannot access static abstract members directly."
```

Comment:
Do you think a new diagnostic should be used, or we can reuse some old one? Like the standard "no method found"? Since non-virtual abstract methods shouldn't even be considered by compiler.

Reply:
I am with Edgar here, more explicit one is better showing what the problem is.

Reply:
Sure, but we shouldn't even resolve such methods, so they're not suggested in autocomplete or anywhere else really.